src/WorkOS.net/Services/SSO/Enums/ConnectionStatus.cs

```diff
@@ -5,10 +5,10 @@
     using Newtonsoft.Json.Converters;
 
     /// <summary>
-    /// An enum describing the linked status of a <see cref="Connection"/>.
+    /// An enum describing the linked state of a <see cref="Connection"/>.
     /// </summary>
     [JsonConverter(typeof(StringEnumConverter))]
-    public enum ConnectionStatus
+    public enum ConnectionState
```

Comment:
We need to change the values as well.

The possible values for `state` are `draft`, `active`, and `inactive`.

Also, I think we should go through a deprecation cycle for languages that support it.

So for .NET we should leave the existing `ConnectionStatus` enum and `Status` property on the `Connection` but mark them both with the [Obsolete attribute](https://docs.microsoft.com/en-us/dotnet/api/system.obsoleteattribute?view=net-5.0), with a message indicating that `State` should be used instead.

This gives developers a chance to transition without breaking the SDK usage immediately.

Reply:
gotcha